src/types/express-serve-static-core.ts

```diff
@@ -0,0 +1,9 @@
+import "express-serve-static-core";
+
+declare module "express-serve-static-core" {
+  interface Request {
+    user?: {
```

Comment:
Does it work properly?
I had some issues with this setup previously((

Reply:
Also consider moving this under express subfolder to index.d.ts:
<img width="187" alt="image" src="https://github.com/user-attachments/assets/d156f80c-0d8d-40ba-a18c-f3515dfd7c19" />